### DISCUSSION TITLE
Atlas For Gamers

### DISCUSSION THREAD

**Discussion by nzmaniac2** [Atlas-OS/Atlas#1187] [Help & Support] [answered]
the current play book 0.4.0 is awesome straight up covers most users who want a faster functioning OS with all the trimmings but as a hard core gamer i prefer to have windows completely gutted no firewall no defender no updates and as many services striped out so my question is can we have a second 0.4.0 playbook with gaming in mind no firewall no uac no updates no edge no defender some things that would be cool in the paly book is option for sniping tool mobile phone found by os when its plugged in an print to pdf and dotnet 3.5. Gut windows one last time for the gamer who has a second dedicated rig like me

**Comment by Ast3risk-ops** [comment]
> the current play book 0.4.0 is awesome straight up covers most users who want a faster functioning OS with all the trimmings but as a hard core gamer i prefer to have windows completely gutted no firewall no defender no updates and as many services striped out so my question is can we have a second 0.4.0 playbook with gaming in mind no firewall no uac no updates no edge no defender some things that would be cool in the paly book is option for sniping tool mobile phone found by os when its plugged in an print to pdf and dotnet 3.5. Gut windows one last time for the gamer who has a second dedicated rig like me

Atlas is meant to not break stuff. As stated in one of our Discord bot autoresponses:

### Why doesn't Atlas do [x] tweak?

Atlas strives to maintain the best compatibility possible alongside performance. We won't disable further processes for a placebo; we'll only configure for a positive impact on Windows's privacy, performance, or security.

### With Atlas, you can be confident in its stability and compatibility, ensuring a smooth experience for everyone.

Windows modifications shouldn't be compared purely based on process count or storage size. Modifications that strip more than Atlas does are likely to break compatibility and cause stability issues.

Yes, but Atlas still could be missing something, right?
Of course! However, remember that there are likely no secret 'magic' Windows tweaks, especially ones that claim to improve performance with almost no downsides. We've filtered out many.

Make a [GitHub Issues](https://github.com/Atlas-OS/Atlas/issues) or a [pull request](https://github.com/Atlas-OS/Atlas/pulls) if you have a good suggestion.

**Comment by pencilnav** [comment, marked as answer]
> can we have a second 0.4.0 playbook with gaming in mind no firewall no uac no updates no edge no defender

The features you've mentioned are enabled by default starting from `v0.3.0`, the reasons are explained [on our docs](https://docs.atlasos.net/general-faq/atlas-and-security/#what-has-changed-since). They can be disabled when applying the Atlas Playbook with AME Wizard, and/or be disabled with the Atlas Desktop Folder.
